backend/tests/v5/unit/models/revisions.test.js

```diff
@@ -53,7 +53,75 @@ const testGetLatestRevision = () => {
 	});
 };
 
+const testGetRevisions = () => {
+	describe('GetRevisions', () => {
+		const expectedData = [
+			{ _id: 1, author: 'someUser', timestamp: new Date() },
+			{ _id: 2, author: 'someUser', timestamp: new Date() },
+			{ _id: 3, author: 'someUser', timestamp: new Date(), void: true },
+		];
+
+		test('Should return non void container revisions', async () => {
```

Comment:
this test is pretty pointless, as the logic of void filtering is done by the db query (and in this case, it is done by the mock value logic here). I think you only really the test on L71

this function basically should return what the database query gives you, so as long as the return value matches what db.find returns this is the only test we need to do.

Reply:
if I totally remove this I get an 'uncovered line' complain for line 62
![image](https://user-images.githubusercontent.com/71635617/134169843-1c9066f1-22f8-4f3e-9d32-ad810b98a8bd.png)